controller/persistence/session_store.go

```diff
@@ -179,7 +179,7 @@ func (store *sessionStoreImpl) initializeLocal() {
 	store.symbolServicePolicies = store.AddFkSetSymbol(FieldSessionServicePolicies, store.stores.servicePolicy)
 	store.AddSymbol(FieldSessionType, ast.NodeTypeString)
 
-	store.AddFkConstraint(store.symbolApiSession, false, boltz.CascadeDelete)
+	//store.AddFkConstraint(store.symbolApiSession, false, boltz.CascadeNone)
```

Comment:
If this is set to `CascadeNone` apiSession deletes will fail w/ a reference error.

Either we just allow any junk data in the field (see session_store_test.go changes) or add a new type of cascade that ignores reference errors on delete.

Reply:
Per conversation add `boltz.CascadeCreateUpdate`